WORKSPACE

```diff
@@ -33,6 +33,17 @@ load("@os_info//:os_info.bzl", "is_linux", "is_windows")
 # bazel dependencies
 haskell_repositories()
 
+http_archive(
```

Comment:
Having to provision tools like that is quite a lot of boilerplate. Especially, if you want to keep snapshot versions in sync. Hazel can use the tools straight from the Stackage snapshot. Is that something that `stack_snapshot` could be taught as well in the future?

Reply:
Well, there's a lot that you can do if you parse `.cabal` files. We don't, and Stack doesn't tell us which dependency is a tool dependency and which one is not.

Reply:
Maybe ultimately a short-coming of stack. Apart from scanning Cabal files, there's also Cabal's [`plan.json`](http://hackage.haskell.org/package/cabal-plan), but last I checked the format wasn't documented and it was unclear if it's stable, yet.